Multisig/App/AppDelegate.swift

```diff
@@ -63,6 +63,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             UINavigationBar.appearance().scrollEdgeAppearance = appearance
         }
 
+        UITableView.appearance().backgroundColor = .backgroundPrimary
```

Comment:
@DmitryBespalov are you fine to use the Appearance protocol?

Reply:
I don't remember having issues in the past except that we need to be aware that there's this global appearance is set when debugging. In the past it was difficult to find why something is having non-standard setting, and it turned out to be because of appearance, which I wasn't aware of. 

So, nothing against it at this point 👍